Cargo.toml

```diff
@@ -18,6 +18,10 @@ repository = "https://github.com/ron-rs/ron"
 documentation = "https://docs.rs/ron/"
 exclude = ["bors.toml", ".travis.yml"]
 
+[features]
+default = ["integer128"]
```

Comment:
Is there a good reason to have it on by default?
I believe generally having less default features is better, unless a feature is *really* expected.

Reply:
While I do expect serde to work for 128b integers, I do agree that non-default features are easier to handle for the end-user.